graphicsmagick.yaml

```diff
@@ -0,0 +1,68 @@
+package:
+  name: graphicsmagick
+  version: "1.3.43"
+  epoch: 0
+  description: GraphicsMagick Image Processing Syste,
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - freetype-dev
+      - libltdl
+      - libpng-dev
+      - libtool
+      - libwebp-dev
+      - libxml2-dev
+      - tiff-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 2b88580732cd7e409d9e22c6116238bef4ae06fcda11451bf33d259f9cbf399f
+      uri: https://sourceforge.net/projects/graphicsmagick/files/graphicsmagick/${{package.version}}/GraphicsMagick-${{package.version}}.tar.xz
```

Comment:
cc @smoser, since this one is new it might be a good candidate for updating swiftly to git-checkout over fetch.